scripts/install_java_linux.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# Install OpenJDK 11.0.2 on Linux
+
+set -eu
+
+# Download and install locations
+JDK_URL=https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz
```

Comment:
I think it would be good to start with the current version of Java (11.0.11). Do we want to update this script for each Java sub version update?

Reply:
I don't see 11.0.11 at https://jdk.java.net/archive/, but I do see it on AdopOpenJDK website ([here][1], which links to [GitHub][2]) and [here][3]. Which one should I use? (I think the last link is the right one but would be good to confirm).

> Do we want to update this script for each Java sub version update?

We can move these links to another file(s) and make `install_java_*.sh` scripts read those file(s). There is a minor issue with Alpine Linux -- it uses [musl libc][musl], not glibc -- so the only way to install OpenJDK there I found was to use its package manager (`apk`).

[1]: https://adoptopenjdk.net/upstream.html?variant=openjdk11&ga=ga
[2]: https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/
[3]: https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/tag/jdk-11.0.11%2B9_openj9-0.26.0
[musl]: https://musl.libc.org/

Reply:
Weird - Yes, maybe I had it on one VM from AdoptOpenJDK. Looking at my dev (macOS) machine I have openJDK installed via brew on 11.0.10. I just wanted to point out that we should include the latest java version available. No idea why the latest in the archive is only 11.0.2.

Reply:
For macOS, we could use Homebrew and the [adoptopenjdk/openjdk](https://github.com/adoptopenjdk/homebrew-openjdk) tap.

Reply:
> For macOS, we could use Homebrew and the [adoptopenjdk/openjdk](https://github.com/adoptopenjdk/homebrew-openjdk) tap.

That is the setup I use personally as it is easy to use and upgrade multiple JDKs I can switch around with `jenv` when needed.